### DISCUSSION TITLE
openai compatible API fails on other local interfaces

### DISCUSSION THREAD

**Discussion by BarfingLemurs** [open-webui/open-webui#1737] [General]
**Is your feature request related to a problem? Please describe.**

TabbyAPI and text-gen-webui are popular interfaces for run performant gpu quantizations, specifically the exl2 quantizations.

I tried running each, text-gen-webui with no API key, and tabbyAPI with its key.

I'm not able to see text-gen-webui models or get something working there, but for tabbyAPI I can see the available models, and can select the current one loaded. However, the response I get it always a single word:

https://imgur.com/a/9hjoRFh

### Info

After login, go to Newchat > settings connections > show openai API http://192.168.1.4:5000/v1 & key

Standard cpu docker installation, then went to localhost:3000

tabbyAPI log: https://pastebin.com/KnBmZtp3

linux, Firefox browser

**Comment by tjbck** [comment]
A

**Comment by tjbck** [comment]
I'm just kidding 😂 That's strange, I'll take a look!

**Comment by justinh-rahb** [comment]
OpenAI API endpoint configurations need to have an API key or it won't activate.
